cache/sampleStoreInit.js

```diff
@@ -18,7 +18,7 @@ const Subject = require('../db').Subject;
 const featureToggles = require('feature-toggles');
 const redisClient = require('./redisCache').client.sampleStore;
 const samsto = require('./sampleStore');
-const log = require('winston');
+const log = require('@salesforce/refocus-logging-client');
```

Comment:
let's switch this var name to "logger" just for consistency across all the other modules